src/cisco_gnmi/nx.py

```diff
@@ -53,11 +57,296 @@ class NXClient(Client):
     >>> print(capabilities)
     """
 
-    def get(self, *args, **kwargs):
-        raise NotImplementedError("Get not yet supported on NX-OS!")
+    def xpath_to_path_elem(self, request):
```

Comment:
From peer review: Might not belong within cisco-gnmi.
Please provide an example `request` to demonstrate what's going on here.
Might be able to be specialized akin to `Client.parse_xpath_to_gnmi_path`.